<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image to PDF Converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mfmcimg2pdf.qrc">
    <normaloff>:/mfmc logo 2015 (square).png</normaloff>:/mfmc logo 2015 (square).png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>580</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="statusTip">
     <string>Exit from program</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_orig">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>441</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>Image File</string>
    </property>
    <widget class="QLabel" name="label_orig_file">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>580</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Name of file to convert</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>fax1.tif</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_orig_img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>551</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Image to convert</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="mfmcimg2pdf.qrc">:/mfmc logo 2015 (square).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_left">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>580</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="mfmcimg2pdf.qrc">
       <normaloff>:/left arrow.png</normaloff>:/left arrow.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_right">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>580</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="mfmcimg2pdf.qrc">
       <normaloff>:/right arrow.png</normaloff>:/right arrow.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_page">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>580</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>   page</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_total_pages">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>580</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>   of 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_current_page">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>580</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <zorder>lineEdit_current_page</zorder>
    <zorder>pushButton_left</zorder>
    <zorder>label_page</zorder>
    <zorder>label_total_pages</zorder>
    <zorder>label_orig_file</zorder>
    <zorder>label_orig_img</zorder>
    <zorder>pushButton_right</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBoxPDF">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Convert to PDF</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>File name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPDF_file">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Name of PDF file to output</string>
     </property>
     <property name="text">
      <string>fax1.pdf</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConvert">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Convert file to PDF</string>
     </property>
     <property name="text">
      <string>Convert and next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSkip">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="statusTip">
      <string>Do not convert this file and skip to next file</string>
     </property>
     <property name="text">
      <string>Skip this file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAll">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="statusTip">
      <string>Convert all files with default names</string>
     </property>
     <property name="whatsThis">
      <string>Convert all files with default names</string>
     </property>
     <property name="text">
      <string>Convert All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNewFiles">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="statusTip">
      <string>Choose new files to convert</string>
     </property>
     <property name="text">
      <string>Choose new files</string>
     </property>
    </widget>
   </widget>
   <zorder>group_orig</zorder>
   <zorder>pushButtonExit</zorder>
   <zorder>groupBoxPDF</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChoose_Files"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChoose_Files">
   <property name="text">
    <string>Choose New Files</string>
   </property>
   <property name="statusTip">
    <string>Choose files to convert</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit Program</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEditPDF_file</tabstop>
  <tabstop>pushButtonConvert</tabstop>
  <tabstop>pushButtonSkip</tabstop>
 </tabstops>
 <resources>
  <include location="mfmcimg2pdf.qrc"/>
 </resources>
 <connections/>
</ui>
